src/identity/PublicIdentity.ts

```diff
@@ -44,7 +51,12 @@ export default class PublicIdentity implements IPublicIdentity {
       return undefined
     }
   }
-
+  /**
+   * [STATIC] Resolve internal and external DIDs from a specific identifier
+   * @param identifier
```

Comment:
* Missing: example and parameters description

Reply:
Found this really difficult to find an example.